modules/openapi-generator/src/main/resources/dart-dio/pubspec.mustache

```diff
@@ -2,8 +2,6 @@ name: {{pubName}}
 version: {{pubVersion}}
 description: {{pubDescription}}
 homepage: {{pubHomepage}}
-authors:
-  - '{{{pubAuthor}}} <{{{pubAuthorEmail}}}>'
```

Comment:
someone recently added this. why removing it?

Reply:
This is deprecated: https://dart.dev/tools/pub/pubspec#authorauthors

Reply:
I can leave it for the old generators but definitely removing it for dart-dio-next.

Reply:
oh didnt know that, funny someone added this in the last month or two